lib/SPIRV/SPIRVReader.cpp

```diff
@@ -4825,6 +4825,137 @@ Instruction *SPIRVToLLVM::transRelational(SPIRVInstruction *I, BasicBlock *BB) {
   return cast<Instruction>(Mutator.getMutated());
 }
 
+std::optional<SPIRVModuleReport> getSpirvReport(std::istream &IS) {
+  int IgnoreErrCode;
```

Comment:
Should this be initialized to something? Thanks

Reply:
No, we don't need its value whatsoever. It's a destination of a value that is always ignored.